source/includes/list-table-shell-properties.rst

```diff
@@ -103,12 +103,17 @@
    * - ``redactHistory``
      - string
      - ``remove``
-     - Controls what information is recorded in the shell history.
-       Must be one of:
+     - Controls what information is recorded in the ``mongosh`` command history.
+       You can access the ``mongosh`` command history via the up and down keys 
+       and Ctrl+R while inside the shell.
```

Comment:
[question/nit] Should we say "or" here since the Ctrl + R search mechanism in mongosh is separate from the up & down key navigation of the command history? 
```suggestion
       or Ctrl+R while inside the shell.
```